mailgun.js

```diff
@@ -0,0 +1,173 @@
+              <p style="font-family: sans-serif; font-size: 14px; font-weight: normal; margin: 0; Margin-bottom: 15px;">Hi there,</p>
+              <p style="font-family: sans-serif; font-size: 14px; font-weight: normal; margin: 0; Margin-bottom: 15px;">`+ toEmail.question+`</p>
+              <table border="0" cellpadding="0" cellspacing="0" class="btn btn-primary" style="border-collapse: separate; mso-table-lspace: 0pt; mso-table-rspace: 0pt; width: 100%; box-sizing: border-box;">
+                <tbody>
+                <tr>
+                  <td align="center" style="font-family: sans-serif; font-size: 14px; vertical-align: top; padding-bottom: 15px;">
+                  <table border="0" cellpadding="0" cellspacing="0" style="border-collapse: separate; mso-table-lspace: 0pt; mso-table-rspace: 0pt; width: auto;">
+                    <tbody>
+                    `+ toEmail.options.map(options => (
+                      `<tr>
+                      <td style="font-family: sans-serif; font-size: 14px; vertical-align: top; background-color: #3498db; border-radius: 5px; text-align: center;"> <a href = "https://polls-angular-ui.herokuapp.com/thankyoupage/?option_id=`+options.option_id+`&poll_id=`+options.poll_id+`" target="_blank" style="display: inline-block; color: #ffffff; background-color: #3498db; border: solid 1px #3498db; border-radius: 5px; box-sizing: border-box; cursor: pointer; text-decoration: none; font-size: 14px; font-weight: bold; margin: 0; padding: 12px 25px; text-transform: capitalize; border-color: #3498db;"> `+options.content+` </a> </td>
+                    </tr>`
+                    ))+`
+                    </tbody>
+                  </table>
+                  </td>
+                </tr>
+                </tbody>
+              </table>
+              </td>
+            </tr>
+            </table>
+          </td>
+          </tr>
+        </table>
+        <div class="footer" style="clear: both; Margin-top: 10px; text-align: center; width: 100%;">
+          <table border="0" cellpadding="0" cellspacing="0" style="border-collapse: separate; mso-table-lspace: 0pt; mso-table-rspace: 0pt; width: 100%;">
+          <tr>
+            <td class="content-block" style="font-family: sans-serif; vertical-align: top; padding-bottom: 10px; padding-top: 10px; font-size: 12px; color: #999999; text-align: center;">
+            <span class="apple-link" style="color: #999999; font-size: 12px; text-align: center;">Turntabl, Achimota, Mile 7, Sonnidom house</span>
+            <br><a href="http://turntabl.io" style="text-decoration: underline; color: #999999; font-size: 12px; text-align: center;">Turntabl</a>.
+            </td>
+          </tr>
+          </table>
+        </div>
+        </div>
+      </td>
+      <td style="font-family: sans-serif; font-size: 14px; vertical-align: top;">&nbsp;</td>
+      </tr>
+    </table>
+    </body>
+  </html>`,
+}, (err, info) => {
+  if (err) {
+    console.log(`Error: ${err}`);
```

Comment:
who are these designed for? start to use the javascript debugger to investigate success and failure of client-side code.